pages/developers/precompiles/features/governance/preimage.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Preimage Precompile Contract
+description: Learn how to take the first necessary step to submit a proposal on-chain by submitting a preimage that contains the action to be carried out in the proposal, using Tangle’s Preimage Precompile.
+---
+
+import GithubFileReaderDisplay from "../../../../../components/GithubFileReaderDisplay";
+
+# Interacting with the Preimage Precompile
+
+## Introduction
+
+As a Polkadot parachain and decentralized network, Tangle features native on-chain governance that enables stakeholders to participate in the direction of the network. With governance v2 (OpenGov), the Substrate Preimage Pallet allows token holders to take the first step toward creating a proposal by submitting the preimage (the action to be carried out in the proposal) on-chain. The hash of the preimage is required to submit the proposal.
```

Comment:
Not correct here @shekohex

Reply:
Addressed in [`89b7660` (#104)](https://github.com/tangle-network/docs/pull/104/commits/89b7660ee4b4383f0bb03f5f82f3f2ade69703ec)